### PR TITLE
add Dockerfile for Ubuntu bionic

### PR DESCRIPTION
**Pull request by slhck**

### CODE REVIEW

**Comment by slhck** [comment]
Hm, the build fails, but not due the Bionic Dockerfile that I added.

File: bionic/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    && localedef -i en_US -f UTF-8 en_US.UTF-8 \
+    && useradd -m -s /bin/bash linuxbrew \
+    && echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
+
+USER linuxbrew
+WORKDIR /home/linuxbrew
+ENV PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH \
+    SHELL=/bin/bash
+
+RUN git clone https://github.com/Linuxbrew/brew.git /home/linuxbrew/.linuxbrew/Homebrew \
```
Comment (maxim-belkin) [inline bionic/Dockerfile]: Homebrew/brew

Comment (slhck) [reply bionic/Dockerfile]: Would it make sense to first merge this and then do a global replacement of those URLs, seeing that they're part of all Dockerfiles?

Comment (maxim-belkin) [reply bionic/Dockerfile]: You're a perfectionist but in a strange way :)
I prefer to merge things that work. If you want, you can do mass fix first and then we'll merge this one but with correct URL

Comment (slhck) [reply bionic/Dockerfile]: Heh, I'm just being economic.
But sure, I'll update it.

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Merged. Thanks again for your contribution to Linuxbrew, Werner!

**Pull request merged**